spec/github_helper_spec.rb

```diff
@@ -2,6 +2,30 @@
 require 'webmock/rspec'
 
 describe Fastlane::Helper::GithubHelper do
+  describe '#initialize' do
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        user: instance_double('User', name: 'test'),
+        'auto_paginate=': nil
+      )
+    end
+
+    before do
+      allow(Octokit::Client).to receive(:new).and_return(client)
+    end
+
+    it 'with the correct github_token' do
+      expect(described_class).to receive(:new).with(github_token: 'GITHUB_TOKEN')
+      described_class.new(github_token: 'GITHUB_TOKEN')
+    end
```

Comment:
This one is a bit too trivial to be useful imho, because you're basically saying "I expect `described_class` to receive `.new(github_token: 'GITHUB_TOKEN')`… and then you proceed to directly do exactly that. So It' like verifying that `a` is `1` right after calling `a=1` 😅 

I think we can omit this specific test then.

Reply:
Yeah, you're right! This does not bring any value for us, I'll remove it. 😄